group/add-member/v2/schema.js

```diff
@@ -35,7 +36,7 @@ module.exports = {
       required: ['group', 'members'],
       properties: {
         group: { $ref: '#/definitions/tangle/update' },
-        members: { $ref: '#/definitions/tangle/any' }
+        members: { $ref: '#/definitions/tangle/update' }
```

Comment:
shouldn't this have an explicit root somewhere though? related https://github.com/ssbc/ssb-tribes2/issues/33

Reply:
It could, currently it's hanging off the group/init which I think is fine.
It all depends on how we define our tangle, and how we define the "reduce" of the info in that tangle.

Gonna merge this for the moment @Powersource